README.md

```diff
@@ -238,6 +240,26 @@ Commands:
     -h, --help  Show this message and exit.
   ```
 
+- `pypi`: scan a pypi package.
+
+  ```
+  ggshield scan pypi PACKAGE_NAME
+  ```
+
+  No options or arguments.
+
+  Under the hood this command uses the `pip download` command to download the python package.
+  You can use `pip` environment variables to set `pip download` parameters as explained in [pip documentation](https://pip.pypa.io/en/stable/topics/configuration/#environment-variables)
+  For example, you can set `pip` `--index-url` parameter by setting `PIP_INDEX_URL` environment variable.
+
+- `archive`: scan an archive files.
```

Comment:
```suggestion
- `archive`: scan an archive file, either a wheel package (`.whl`) or source distribution archive (`.tar.gz`)
```

Reply:
The `scan pypi` should be used for that. `shutils.unpack_archive` does not handle wheel format.